subsystem-drive/implementations/rules-engine.hpp

```diff
@@ -10,6 +11,16 @@ namespace sjsu::drive
         static constexpr int kMaxSpeed = 100;
         drive_commands ValidateCommands(drive_commands commands)
         {
+
+             if(commands.mode == 'D')
+            {
+                if(commands.angle > clamp_angle || commands.angle < -clamp_angle)
+                {
+                    commands.angle = std::clamp(commands.angle, -clamp_angle, clamp_angle);
+                    //sjsu::LogInfo("Angle:", commands.angle);
+                    //sjsu::LogInfo("Specified angle is too large... clamping angle");
+                }
```

Comment:
I think we should remove the if statement since std::clamp runs this internally most likely